doc/getting-started/building-the-node-on-windows.md

```diff
@@ -0,0 +1,110 @@
+# Development build of Cardano Node on Windows
+
+This document describes how to build a __DEVELOPMENT__ version of `cardano-node`.
+This is not a __PRODUCTION__ build!
+
+Building on `Windows` is a non trivial task.  There are various tools one needs
+to install and configure.  We recommend to use
+[chocolatey](https://chocolatey.org) (which provides the `choco` command) to
+install them.
+
+All instructrions that invoke `choco` can be done in a PowerShell with root
+previlages.  For development work we recommend installing and using `git-bash`
+(with `choco install git-bash`).
+
+## Install GHC
+
+The recommended way is to
+
+```PowerShell
+choco install --version 8.6.5 ghc
+```
+in PowerShell>
+
+We are about to transition to `ghc-8.10.2` but this version is broken on
+windows ([see](https://gitlab.haskell.org/ghc/ghc/-/issues/18550)), though
+a fix is rather easy.
+
+## Install pkg-config
+
+```PowerShell
+choco install pgkconfiglite
```

Comment:
This also results in an error:
`PS C:\WINDOWS\system32> choco install pgkconfiglite
Chocolatey v0.10.15
Installing the following packages:
pgkconfiglite
By installing you accept licenses for the packages.
pgkconfiglite not installed. The package was not found with the source(s) listed.
 Source(s): 'https://chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://chocolatey.org/docs/troubleshooting for more
 assistance.`

Reply:
This works for me:
```
choco search pkgconfiglite
...
pkgconfiglite 0.28
1 packages found.
```